src/components/Stepper/Stepper.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { FC } from "react";
+
+import { BaseStepperProps } from "./type";
+import { StyledStepper } from "./style";
+import PlusIcon from "../../assets/icon-plus.svg";
+import MinusIcon from "../../assets/icon-minus.svg";
+
+const Stepper: FC<BaseStepperProps> = (props) => {
+  const { onAddClick, onRemoveClick, quantity } = props;
+
+  return (
+    <StyledStepper>
+      <div className="StepperWrapper">
+        <button className="butttonWrapper" onClick={() => onRemoveClick()}>
+          <img src={MinusIcon} className="iconWrapper" />
```

Comment:
before using this method i use that  { ReactComponent as MinusIcon } and pass it as a component, but there is a styling-issue since class iconWrapper can not applied to the Component

Reply:
oh really? ga bisanya gara-gara error typescript atau gimana?

Reply:
gamasuk gitu style nya, lemme try again ya

Reply:
dah gue fixed ya ini, masalah webpack tadi harus pass exportType: 'named'

Reply:
done